module/platform/OpenCR/HardwareIO/src/hardwareio/process_data.cpp

```diff
@@ -132,16 +132,16 @@ namespace module::platform::OpenCR {
         servo_states[servo_index].temperature = convert::temperature(data.present_temperature);
 
         // Buzz if any servo is hot, use the boolean flag to turn the buzzer off once the servo is no longer hot
-        bool servo_hot = false;
+        bool any_servo_hot = false;
         for (const auto& servo : servo_states) {
             if (servo.temperature > cfg.alarms.temperature.level) {
-                servo_hot = true;
+                any_servo_hot = true;
                 emit(std::make_unique<Buzzer>(cfg.alarms.temperature.buzzer_frequency));
                 break;
             }
         }
```

Comment:
There's a thing for this called `std::any_of` you could use instead of this

Reply:
Somebody (I can't remember who) requested for this part to be able to spit out the ID(s) of hot servo(s). I will be changing this part in a future pull request